lib/src/utils/timer.dart

```diff
@@ -109,8 +94,8 @@ class DsTimer {
         _functionsMap.remove(fun);
         try{
           fun();
-        } catch(err,stack) {
-          print("callback error; $err\n$stack");
+        } catch(err, stack) {
+          logger.warning("Timer callback error.", err, stack);
```

Comment:
Showing errors using print() circumvents the logger, which is usually not desired.